docs/toc.md

```diff
@@ -79,6 +79,7 @@
 ## [Rotate](animations/Rotate.md)
 ## [Saturation](animations/Saturation.md)
 ## [Scale](animations/Scale.md)
+## [ViewportBehavior](animations/ViewportBehavior.md)

```

Comment:
I think this should go in the helpers as it's not an animation

Reply:
Done, move to helpers folder.